drivers/storage/ide/uniata/id_init.cpp

```diff
@@ -2555,7 +2555,7 @@ AtapiChipInit(
                 * size goes below 32DW.  Setting it to 1 makes the watermark
                 * 64DW.
                 *
-                * http://www.reactos.org/bugzilla/show_bug.cgi?id=6500
+                * https://reactos.org/bugzilla/show_bug.cgi?id=6500
```

Comment:
Broken link.
Seems to be https://jira.reactos.org/browse/CORE-5897

```suggestion
                * https://reactos.org/bugzilla/show_bug.cgi?id=6500
                * https://jira.reactos.org/browse/CORE-5897
```

Reply:
I do hope so, because we are beginning to become quite offtopic. :-)